packages/types/types/index.d.ts

```diff
@@ -147,6 +154,38 @@ export interface ThemeProviderProps {
   customBreakpoints?: string[]
 }
 
+export interface ColorValidation {
+  color?: PropValidator
+}
+
+export interface CheckboxProps extends DsRefProps, ColorValidation {
+  id: string
+  size?: number
+  onChange: () => void
+}
+
+export interface FormFieldProps {
+  children?: PropValidator
```

Comment:
It should be safe to omit `children` since that's already taken care of when extending `React.Component`